main.py

```diff
@@ -231,6 +230,38 @@ def kcidb_send_notification(data, context):
     message = SPOOL_CLIENT.pick(notification_id)
     if not message:
         return
+    # log message before sending email
```

Comment:
We're not logging **the** message here, not the notification's message, that is. We're only logging the notification's ID.

In some sense we **do** log a "message", that is "SENDING foobar", but that hardly needs a comment. If we have to have a comment, let it at least be something like "Log the notification ID", even though that is quite obvious from the code, but if you feel that's needed, that's fine too.

So, please either remove this comment, or change it to something like "Log the notification ID", whichever you prefer. Please also do the same to the similar comment below.